### DISCUSSION TITLE
```net8.0``` ```NETSDK1206``` RID win10-arm64, win10-x64, win10-x86 warning

### DISCUSSION THREAD

**Discussion by limeniye** [unoplatform/uno#15882] [General]
Today I found the following warning.

![image](https://github.com/unoplatform/uno/assets/69314237/cf5198ea-372f-48e4-85a3-4b41743bc48a)

```
1>C:\Program Files\dotnet\sdk\8.0.201\Sdks\Microsoft.NET.Sdk\targets\Microsoft.NET.Sdk.targets(284,5): warning NETSDK1206: Found version-specific or distribution-specific runtime identifier(s): win10-arm64, win10-x64, win10-x86. Affected libraries: Microsoft.WindowsAppSDK. In .NET 8.0 and higher, assets for version-specific and distribution-specific runtime identifiers will not be found by default. See https://aka.ms/dotnet/rid-usage for details.
```

I'm not sure if this is normal compilation behavior.
And I see that the compilation was successful. 
But I am concerned about this message.
Also I've seen a similar effect here: https://github.com/unoplatform/uno/discussions/15262#discussioncomment-8553975

**Comment by spouliot** [comment]
It's a warning, no need for concern. The RID have been renamed to `win-{arch}`. IOW MS removed the version `10` which was unchanged for Windows 11 anyway.

You can edit your project to remove the `10` from the RID and build again, without the warnings.

Is this a new project ? or something you already had created a while ago ?

I know the templates have been updated to reflect this but maybe we missed a project type ?

**Reply by limeniye** [reply]
I created this project a couple hours ago.

**Reply by spouliot** [reply]
c.c. @nickrandolph

**Reply by limeniye** [reply]
Opened issue: https://github.com/unoplatform/uno/issues/15884
